content/en/getting_started/session_replay/_index.md

```diff
@@ -0,0 +1,140 @@
+
+From this page, you can also expand the Performance Waterfall by selecting **Expand Waterfall** for more detailed information.
+
+{{< img src="/getting_started/session_replay/performance-waterfall.png" alt="Expanded performance waterfall">}}
+
+## Troubleshoot using Developer Tools
+
+Session Replay's [Browser Developer Tools][8] enable you to view various aspects of your application's performance, console logs and errors associated with the view, and any application or user attributes regarding the replay. 
+
+{{< img src="/getting_started/session_replay/dev-tools.png" alt="Dev tools console" style="width:100%" >}}
+
+You can view any **Resources** associated with a view by selecting the Resources tab.
+
+{{< img src="/getting_started/session_replay/resources.png" alt="Resources associated with the view" style="width:100%" >}}
+
+**Traces** associated with a view can be found in the [Traces tab](#tracing-with-the-apm-integration).
+
+{{< img src="/getting_started/session_replay/traces.png" alt="Traces associated with the view" style="width:100%" >}}
+
+Session metadata is populated in the **Attributes** tab.
+
+{{< img src="/getting_started/session_replay/attributes.png" alt="Attributes tab of the Dev tools console" style="width:100%" >}}
+
+## Pivot to correlated data
+
+Session Replay integrates with your application's metrics, traces, and logs to give you helpful context for debugging issues. Utilizing the APM and Error Tracking integrations alongside Session Replay enables you to investigate the root cause of user-facing issues, regardless of where they originate in your stack.
+
+### Tracing with the APM integration
+
+Combining Session Replay with [APM traces][9] enables you to receive end-to-end visibility across frontend and backend issues, and see how code and infrastructure are impacting your user experience.
+
+Within the Traces tab, select **View Trace in APM** to see more detailed information, including errors and logs associated with the trace. 
+
+{{< img src="/getting_started/session_replay/APM.png" alt="APM page with more detailed information" style="width:100%" >}}
+
+### Investigating errors with the Error Tracking integration
+
+The [Error Tracking integration][10] is helpful in debugging issues and getting to the root cause. You can receive alerts for an error, see the exact line of code that caused it, and pivot to view a user session that encountered the error on the Datadog site.
+
+In the **Errors** tab, selecting a specific error expands the view to display the time the error occurred, as well as the error message. Clicking **Issue in error tracking** opens a panel with more detail surrounding the error, as well as any attributes associated with the session.
+
+{{< img src="/getting_started/session_replay/error-tracking.png" alt="Error tracking panel" style="width:100%" >}}
+
+## What's next?
+
+### Create Browser Tests from Session Replays
```

Comment:
```suggestion
### Create Synthetic browser tests from Session Replays
```
The Synthetic part is important -- Synthetic tests are automated tests that Datadog simulates on real running code.